build/release/common.sh

```diff
@@ -133,7 +133,7 @@ github_release_complete() {
 write_version_file() {
     # upload a file with the version number
     cat <<EOF > ${RELEASE_DIR}/version
-${version}
```

Comment:
it think it was previously incorrect, we had to of  lot of trial and error to pull the version out and pass it on to e2e test, basam helped us with these changes and the build and e2e test are working now.